tests/test_crash_reports.py

```diff
@@ -193,18 +185,18 @@ def test_that_only_plugin_reports_have_plugin_icon(self, mozwebqa):
         reports_page.click_filter_by(type)
         reports_page.click_filter_days_by(days)
         reports_page.click_filter_os_by(os)
-        Assert.equal((type, days, os), (reports_page.current_filter_type,
-                                        reports_page.current_days_filter,
-                                        reports_page.current_os_filter))
+        assert (type, days, os) == (reports_page.current_filter_type,
+                                    reports_page.current_days_filter,
+                                    reports_page.current_os_filter)
```

Comment:
As above re: `type` and splitting into separate lines. As `type` appears now in at least two tests we could leave it as a later fix.